config/cmake/libh5cc.in

```diff
@@ -43,8 +44,16 @@ for arg in $@ ; do
       showconfigure
       exit $status
       ;;
+    -show)
```

Comment:
still missing:
-help | --help | -h   This help message.
-prefix=DIR             Prefix directory to find HDF5 lib/ and include/
-shlib                Compile with shared HDF5 libraries [default for hdf5 built
                                                     without static libraries]
-noshlib              Compile with static HDF5 libraries [default for hdf5 built
                                                     with static libraries]
-echo                 Show all the shell commands executed

Reply:
Other information displayed by help:
```
  <compile line>  - the normal compile line options for your compiler.
                    h5pcc uses the same compiler you used to compile
                    HDF5. Check with your compiler's man pages for more
                    information on which options are needed.
 
 You can override the compiler, linker, and whether or not to use static
 or shared libraries to compile your program by setting the following
 environment variables accordingly:
 
   HDF5_CC                  -  use a different C compiler
   HDF5_CLINKER             -  use a different linker
   HDF5_USE_SHLIB=[yes|no]  -  use shared or static version of the HDF5 library
                                 [default: no except when built with only
                                           shared libraries]
 
 You can also add or change paths and flags to the compile line using
 the following environment variables or by assigning them to their counterparts
 in the 'Things You Can Modify to Override...' section of h5pcc
 
  Variable              Current value to be replaced
  HDF5_CPPFLAGS         ""
  HDF5_CFLAGS           ""
  HDF5_LDFLAGS          ""
  HDF5_LIBS             ""
 
 Note that adding library paths to HDF5_LDFLAGS where another hdf5 version
```

Reply:
I'm not sure it is an autotools thing; HDF5 provides a script to compile an HDF5 program, and a user will want to know how to use it and what options can be given. So, how should a user specify compiling an HDF5 program and only use the static version in the cmake version of h5cc?